uaclient/api/u/pro/security_status/packages/v1.py

```diff
@@ -0,0 +1,57 @@
+from apt import Cache  # type: ignore
+
+from uaclient import system
+from uaclient.api.api import APIEndpoint
+from uaclient.api.data_types import AdditionalInfo
+from uaclient.config import UAConfig
+from uaclient.data_types import DataObject, Field, StringDataValue
+
+
+class InstalledPackagesResults(DataObject, AdditionalInfo):
+    fields = [
+        Field("manifest_data", StringDataValue),
+    ]
+
+    def __init__(self, manifest_data: str):
+        self.manifest_data = manifest_data
+
+
+def get_installed_packages() -> InstalledPackagesResults:
+    return _get_installed_packages(UAConfig())
+
+
+def _get_installed_packages(cfg: UAConfig) -> InstalledPackagesResults:
+    """Returns the status of installed packages (apt and snap packages)
+    Returns a string in manifest format i.e. package_name\tversion
+    """
+    cache = Cache()
+    manifest = ""
+    # get snap packages
+    out, _ = system.subp(["snap", "list"])
+    apps = out.splitlines()
+    apps = apps[1:]
+    for line in apps:
+        pkg = line.split()
+        manifest += "{app}\t{version}\n".format(app=pkg[0], version=pkg[1])
```

Comment:
snaps should appear at the end of the file in the format 
```
snap:core20	stable	1587
snap:snapd	stable	16292
snap:lxd	4.0/stable/ubuntu-20.04	22753
```

(example taken from https://cloud-images.ubuntu.com/releases/focal/release/ubuntu-20.04-server-cloudimg-amd64-root.manifest)

Reply:
@CalvoM this comment is still valid